src/app/submission/components/metadata-list/metadata-list.component.ts

```diff
@@ -162,7 +175,14 @@ export class MetadataListComponent implements OnInit, OnDestroy {
   }
 
   filterByState(event) {
-    this.dataSource.filterByState(event.value);
+    if(event.value === METADATA_VALIDATION_STATES.GraphInvalid) {
+      // This can be removed in dcp-546
+      // Currently there is no graph invalid state in core
```

Comment:
just exclude the invalid graph state for now in filter options instead of including it and not doing anything

Reply:
You're right, done :)